bs-react-native-jsx3-compat/src/components/button.re

```diff
@@ -9,14 +8,11 @@ let make =
       ~testID=?,
       ~title,
     ) =>
-  ReasonReact.wrapJsForReason(
-    ~reactClass=button,
-    ~props={
-      "accessibilityLabel": accessibilityLabel,
-      "color": color,
-      "disabled": disabled,
-      "onPress": onPress,
-      "testID": testID,
-      "title": title,
-    },
-  );
+  <ReactNative.Button
+    ?accessibilityLabel
+    ?color
+    ?disabled
+    onPress={_ => onPress()}
```

Comment:
Shouldn't be `onPress={onPress)}`? To pass everything that comes from RN?

Reply:
Nope because current bindings doesn't tell you there is an even as an argument. That's why I used this kind of tricks https://github.com/reasonml-community/bs-react-native/pull/472/files#diff-e9317c6473601bfdc17cac2811383838R128

Reply:
Ok, make sense! Thanks for the explanation.